registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/MosipVerticleAPIManager.java

```diff
@@ -169,7 +168,12 @@ public void configureHealthCheckEndpoint(Vertx vertx, Router router, final Strin
 	private boolean checkServletPathContainsCoreProcessor(String servletPath) {
 		return servletPath.contains("packetvalidator") || servletPath.contains("osi") || servletPath.contains("demo")
 				|| servletPath.contains("bio") || servletPath.contains("uin") || servletPath.contains("quality")
-				|| servletPath.contains("abishandler") || servletPath.contains("securezone");
+				|| servletPath.contains("abishandler") || servletPath.contains("securezone")
+				|| servletPath.contains("print") || servletPath.contains("cmd") || servletPath.contains("operator")
+				|| servletPath.contains("supervisor") || servletPath.contains("introducer")
+				|| servletPath.contains("final") || servletPath.contains("biometric")
+				|| servletPath.contains("packetclassifier") || servletPath.contains("bioauth")
+				|| servletPath.contains("external");

```

Comment:
the methods calls out core processor, but not all items mentioned is part of core processor

Reply:
This is an existing method,I renamed as checkServletPath